jekyll/_cci2/templates/template-how-to.adoc

```diff
@@ -0,0 +1,179 @@
+---
+# These platform content tags control the tags that appear at the top of the docs page. Use them to show the reader which CircleCI platforms support the feature or task you are writing about
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+// The page title for a how-to guide should be concise yet descriptive. It immediately tells the reader at a glance what will be accomplished.
+= Page title
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+////
+Some notes on attributes
+
+:page-liquid: - ensures that all liquid tags are processed before rendering the content. More info here: https://github.com/asciidoctor/jekyll-asciidoc/blob/89b8f55f5312e4a0f1bca496bd9217d53d5b21dd/docs/modules/ROOT/pages/liquid.adoc
+
+:icons: font - enables the use of font awesome icons https://docs.asciidoctor.org/asciidoc/latest/macros/icons-font/
+
+:experimental: allows access to asciidoc macros, more info here: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/
+
+////
+
+This template will help you create a new docs page that shows the reader how to carry out a task and accomplish a specific goal with CircleCI. Content templates help you to:
+
+* Develop new content quickly
+* Ensure your page conforms to the style guide
+
+To use this template, make a copy and place it in the `_cci2` directory. The filename should match the page title. Look at the source file for this page template link:https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/template/template-how-to.adoc?plain=1[here]. There is additional detail in the comments.
+
+The opening paragraph should be used to briefly describe to the reader what the reader will do using this guide, and what the outcomes are when they successfully finish the task.
```

Comment:
@rosieyohannan This is where I chose to add the "what the reader will do and achieve" as opposed to further down in the Intro section. My reasoning was I think the reader will want to be able to tell from the first few lines what the goal is for the page. And then I think in the Intro the context for the task (what value it can provide, etc) can be explained a bit more. Thoughts?

Reply:
TBH thinking about this more, I feel like maybe we don't need the intro in how-to guides, telling the reader what they will do and why they should care can all be done in the opening paragraph and then get straight to the prerequs and steps? We can suggest linking to the feature overview page for further reading in this opening para too?